docs/connectors/matrix.md

```diff
@@ -57,8 +57,12 @@ connectors:
 
 ## End to End Encryption
 
+> **Note** - `mxid` & `password` must be used for E2EE to work
```

Comment:
There's a way to do a proper little note box in sphinx but I don't know the MyST syntax for it.

In rst it's `.. note:`

Reply:
Looks like it would be:

```
\`\`\`note
Note contents
\`\`\`
```

https://myst-parser.readthedocs.io/en/latest/syntax/syntax.html#directives-a-block-level-extension-point